scripts/compile-references.js

```diff
@@ -1,3 +1,6 @@
+// @ts-check
```

Comment:
we should have copyright here?

Reply:
ah, see it is below, not sure what it is correct, but anyway

Reply:
`// @ts-check` has to be at the top for it to work apparently